src/DayDifference.php

```diff
@@ -33,14 +35,24 @@ public function __construct(
         DateTimeInterface $startDate,
         DateTimeInterface $endDate,
         array $allowedDaysOfTheWeek = [0, 1, 2, 3, 4, 5, 6],
-        array $excludedDates = []
+        array $excludedDates = [],
     ) {
         $this->startDate = $startDate;
         $this->endDate = $endDate;
         $this->allowedDaysOfTheWeek = $allowedDaysOfTheWeek;
         $this->excludedDates = $excludedDates;
     }
 
+    static function set_cache($cache)
+    {
+        self::$cache = $cache;
+    }
+
+    static function get_cache()
+    {
+        return self::$cache;
+    }
+
```

Comment:
Can we add types to the function match the rest of the file?

Reply:
It has types on the latest commit. I guess i need to organize my commits better.